<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1614871146111" stop="1614871154308" version="1.5.4" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default suite : Default test</name>
    <title>Default suite : Default test</title>
    <test-cases>
        <test-case start="1614871153890" stop="1614871153897" status="broken">
            <name>beforeClass</name>
            <description>@BeforeClass</description>
            <failure>
                <message>NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=89.0.4389.72)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-A4GJKOM', ip: '192.168.1.98', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_271'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 89.0.4389.72, chrome: {chromedriverVersion: 89.0.4389.23 (61b08ee2c5002..., userDataDir: C:\Users\sukki\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:61102}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 6cfde3c487c68dbc0a79e84aab90ed9b</message>
                <stack-trace>org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=89.0.4389.72)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-A4GJKOM', ip: '192.168.1.98', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_271'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 89.0.4389.72, chrome: {chromedriverVersion: 89.0.4389.23 (61b08ee2c5002..., userDataDir: C:\Users\sukki\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:61102}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 6cfde3c487c68dbc0a79e84aab90ed9b&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#13;
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)&#13;
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)&#13;
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)&#13;
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)&#13;
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)&#13;
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions$RemoteWindow.maximize(RemoteWebDriver.java:837)&#13;
	at com.sailotech.testautomation.ezscheduler.accerlators.TestBase.startSession(TestBase.java:213)&#13;
	at com.sailotech.testautomation.ezscheduler.accerlators.TestBase.beforeClass(TestBase.java:51)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)&#13;
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)&#13;
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1614871154155" stop="1614871154157" status="canceled">
            <name>display</name>
            <description>@BeforeClass</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onConfigurationSkip(AllureTestListener.java:120)&#13;
	at org.testng.internal.TestListenerHelper.runPostConfigurationListeners(TestListenerHelper.java:37)&#13;
	at org.testng.internal.Invoker.runConfigurationListeners(Invoker.java:1384)&#13;
	at org.testng.internal.Invoker.handleConfigurationSkip(Invoker.java:250)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:200)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)&#13;
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1614871154159" stop="1614871154159" status="canceled">
            <name>screenRecord</name>
            <description>@BeforeClass</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onConfigurationSkip(AllureTestListener.java:120)&#13;
	at org.testng.internal.TestListenerHelper.runPostConfigurationListeners(TestListenerHelper.java:37)&#13;
	at org.testng.internal.Invoker.runConfigurationListeners(Invoker.java:1384)&#13;
	at org.testng.internal.Invoker.handleConfigurationSkip(Invoker.java:250)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:200)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)&#13;
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1614871154183" stop="1614871154183" status="canceled">
            <name>setUp</name>
            <description>@BeforeMethod</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onConfigurationSkip(AllureTestListener.java:120)&#13;
	at org.testng.internal.TestListenerHelper.runPostConfigurationListeners(TestListenerHelper.java:37)&#13;
	at org.testng.internal.Invoker.runConfigurationListeners(Invoker.java:1384)&#13;
	at org.testng.internal.Invoker.handleConfigurationSkip(Invoker.java:250)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:200)&#13;
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)&#13;
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1614871154227" stop="1614871154228" status="canceled">
            <name>TC_07_ProviderCancelAn_AppointmentsOrSlots</name>
            <description type="text">Provider Cancel or block all the slots</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:186)&#13;
	at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:61)&#13;
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1389)&#13;
	at org.testng.internal.Invoker.registerSkippedTestResult(Invoker.java:1063)&#13;
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:529)&#13;
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.sailotech.testautomation.ezscheduler.tests.Providerrcancelappointments_test"/>
                <label name="testMethod" value="TC_07_ProviderCancelAn_AppointmentsOrSlots"/>
                <label name="severity" value="normal"/>
                <label name="testId" value="TC_07"/>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1614871154237" stop="1614871154238" status="canceled">
            <name>screenShot</name>
            <description>@AfterMethod</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onConfigurationSkip(AllureTestListener.java:120)&#13;
	at org.testng.internal.TestListenerHelper.runPostConfigurationListeners(TestListenerHelper.java:37)&#13;
	at org.testng.internal.Invoker.runConfigurationListeners(Invoker.java:1384)&#13;
	at org.testng.internal.Invoker.handleConfigurationSkip(Invoker.java:250)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:200)&#13;
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:536)&#13;
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1614871154244" stop="1614871154245" status="canceled">
            <name>TC_04_ProviderLogout</name>
            <description type="text">Provider Logout from Ezscheduler Website</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:186)&#13;
	at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:61)&#13;
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1389)&#13;
	at org.testng.internal.Invoker.registerSkippedTestResult(Invoker.java:1063)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.sailotech.testautomation.ezscheduler.tests.Providerrcancelappointments_test"/>
                <label name="testMethod" value="TC_04_ProviderLogout"/>
                <label name="severity" value="normal"/>
                <label name="testId" value="TC_04"/>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1614871154250" stop="1614871154250" status="canceled">
            <name>afterClass</name>
            <description>@AfterClass</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onConfigurationSkip(AllureTestListener.java:120)&#13;
	at org.testng.internal.TestListenerHelper.runPostConfigurationListeners(TestListenerHelper.java:37)&#13;
	at org.testng.internal.Invoker.runConfigurationListeners(Invoker.java:1384)&#13;
	at org.testng.internal.Invoker.handleConfigurationSkip(Invoker.java:250)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:200)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)&#13;
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:214)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1614871154252" stop="1614871154252" status="canceled">
            <name>stopRecording</name>
            <description>@AfterTest</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onConfigurationSkip(AllureTestListener.java:120)&#13;
	at org.testng.internal.TestListenerHelper.runPostConfigurationListeners(TestListenerHelper.java:37)&#13;
	at org.testng.internal.Invoker.runConfigurationListeners(Invoker.java:1384)&#13;
	at org.testng.internal.Invoker.handleConfigurationSkip(Invoker.java:250)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:200)&#13;
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)&#13;
	at org.testng.TestRunner.afterRun(TestRunner.java:725)&#13;
	at org.testng.TestRunner.run(TestRunner.java:509)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="18708@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
